components/nifosg/particle.hpp

```diff
@@ -30,11 +30,13 @@ namespace NifOsg
         ParticleSystem();
         ParticleSystem(const ParticleSystem& copy, const osg::CopyOp& copyop);
 
-        META_Object(NifOsg, NifOsg::ParticleSystem)
```

Comment:
Correct. We can cherry-pick these to master if that makes it easier for you to maintain the patches.